nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java

```diff
@@ -221,6 +224,41 @@ public NullnessStore forceRunOnMethod(TreePath methodPath, Context context) {
     return dataFlow.finalResult(methodPath, context, nullnessPropagation);
   }
 
+  /**
+   * Nullness of the optional content of the expression is checked in the store.
+   *
+   * @param exprPath tree path of the expression
+   * @param context Javac context
+   * @return nullness info of optional content of the optional expression
+   */
+  public Nullness getNullnessOfExpressionOptionalContent(TreePath exprPath, Context context) {
+    NullnessStore store = dataFlow.resultBeforeExpr(exprPath, context, nullnessPropagation);
+
+    // We use the CFG to get the Node corresponding to the expression
+    Set<Node> exprNodes =
+        dataFlow
+            .getControlFlowGraph(exprPath, context, nullnessPropagation)
+            .getNodesCorrespondingToTree(exprPath.getLeaf());
+
+    if (exprNodes.size() != 1) {
+      // Since the expression must have a single corresponding node
+      // NULLABLE is our default assumption
+      return Nullness.NULLABLE;
+    }
+
+    AccessPath ap = AccessPath.fromBaseAndElement(exprNodes.iterator().next(), OPTIONAL_CONTENT);
```

Comment:
Could we make the field name (e.g. `OPTIONAL_CONTENT` a parameter to this method? `getNullnessOfNamedField(...)` for example? 

That would be:
a) More general
b) A way to further decouple the dataflow class from the optional values handler